playground/mocks/config/responseConfig.js

```diff
@@ -33,7 +33,7 @@ const idx = {
     // 'error-identify-with-only-one-third-party-idp',
     // 'authenticator-enroll-email',
     // 'error-internal-server-error',
-    // 'authenticator-enroll-password',
+    'authenticator-enroll-password-with-ad-req',
```

Comment:
Mind un-commenting "identify" above and commenting this one out? We usually don't commit changes that affect the mock response cycle.

```suggestion
    // 'authenticator-enroll-password-with-ad-req',
```

Reply:
Oh yeah this was accidental, thanks for catching it